x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/overview/overview.tsx

```diff
@@ -44,18 +65,45 @@ export const Overview = () => {
     />
   );
 
+  const shouldShowCallout = () => {
+    if (dataStreamsStatus !== 'success') {
+      return false;
+    }
+
+    const { type } = asset;
+    const baseCondition =
+      !isMetricsSignal(dataStreams) && !dismissedAddMetricsCallout && renderMode.mode === 'page';
+
+    const isRelevantContainer =
```

Comment:
```suggestion
    const isContainer =
```

Reply:
I chose to use `isRelevantContainer` because `isContainer` suggests it only needs to match `asset.type === 'container'`, which isn't accurate.

Does that make sense to you?